pkg/bootstrap/docker/up.go

```diff
@@ -334,8 +335,18 @@ func (c *ClientStartConfig) GetDockerClient(out io.Writer) error {
 	if err != nil {
 		return errors.ErrNoDockerClient(err)
 	}
+	// FIXME: Workaround for docker engine API client on OS X - sets the default to
+	// the wrong DOCKER_HOST string
+	if runtime.GOOS == "darwin" {
+		dockerHost := os.Getenv("DOCKER_HOST")
+		if len(dockerHost) == 0 {
+			os.Setenv("DOCKER_HOST", "unix:///var/run/docker.sock")
```

Comment:
No, it'll try to open the socket and say it can't... just like when you have docker machine but don't have the environment configured.